Classes/Common/Helper.php

```diff
@@ -208,6 +208,12 @@ public static function decrypt($encrypted)
      */
     public static function getXmlFileAsString($content)
     {
+        // Don't make simplexml_load_string throw (when $content is an array
+        // or object)
+        if (!is_string($content)) {
```

Comment:
Here could be fixed the PHPDoc, curently it states: _Get content of XML file as string or false if there is nothing._ In fact it is _XML file as SimpleXMLElement_ and _@return mixed_ could be changed to _@return SimpleXMLElement|false_.
 
In general I really like your idea to introduce automatic testing! It is really cool!

Reply:
Yes, absolutely! I updated the PHPDoc.